gym/utils/play.py

```diff
@@ -1,4 +1,7 @@
 """Utilities of visualising an environment."""
+
+# TODO: Convert to new step API in 1.0
```

Comment:
Is this TODO still to do

Reply:
It was too much of a pain to make this compatible with both APIs, so just added this to TODO when we fully switch to new API (I will add a warning though)